examples/example3/nautobot_adapter.py

```diff
@@ -0,0 +1,167 @@
+import os
+import pynautobot
+
+from diffsync import DiffSync
+from models import Region, Country
+
+NAUTOBOT_URL = os.getenv("NAUTOBOT_URL", "https://demo.nautobot.com")
+NAUTOBOT_TOKEN = os.getenv("NAUTOBOT_TOKEN", "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa")
+
+
+class NautobotRegion(Region):
+    """Extend the Region object to store Nautobot specific information.
+
+    Region are represented in Nautobot as a dcim.region object without parent.
+    """
+
+    remote_id: str
+    """Store the nautobot uuid in the object to allow update and delete of existing object."""
+
```

Comment:
This reminds me that we need a pattern for models that do not support CRUD operations - right now it's more than a bit confusing as DiffSync will log messages *saying* that it's performing create/update/delete, it's just actually a no-op that isn't actually changing any underlying system. 

Maybe for now create stub CRUD methods on this model that just log a message saying "Nautobot is not being updated"?